Formula/docksal.rb

```diff
@@ -0,0 +1,25 @@
+class Docksal < Formula
+  desc "Docker powered web development environments for macOS, Windows & Linux"
+  homepage "https://docksal.io"
+  url "https://github.com/docksal/docksal/archive/v1.12.3.tar.gz"
+  sha256 "252a8ce0194fbcacb0a2b7dd6d578843f4ada2cd31c1859350a53791827932a0"
+
+  def install
+    mkdir_p "~/.docksal"
```

Comment:
Do not install things in the user's `HOME`.

Reply:
@fxcoudert thanks for taking the time to look this PR over. Maybe I missed something on the cookbook, but what is the general approach to handling configuration files that are typically put in the user directory?

Reply:
System-wide configuration should be put in `etc`, user-specific configuration should be left to each individual user to choose :)